src/redux/messageReducer.js

```diff
@@ -0,0 +1,36 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  message_data: '',
+  status: '',
+  error: null,
+};
+
+const FETCH_URL = 'http://localhost:3000/api/v1/messages';
+
```

Comment:
- [ ] Kindly correct your fetch_url to the one that generates the random greetings in your backend. The correct url should `'http://localhost:3000/api/v1/messages/index'` based on what I saw in your route and what I used to test your api in the rails-backend app.

![your-backend](https://github.com/Tekhlay/hello-react-front-end/assets/26597432/dd73ab7a-dab0-4e81-a7f3-78f824f541fe)